pkgs/tools/misc/archi/default.nix

```diff
@@ -43,6 +46,7 @@ stdenv.mkDerivation rec {
 
         install -D -m755 Archi $out/libexec/Archi
         makeWrapper $out/libexec/Archi $out/bin/Archi \
+          --prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath ([ webkitgtk ])} \
```

Comment:
Can we rather patchelf?

Reply:
Are you sure that patchelf is viable here?
This package is quite complex, i fear we'd have to change a lot.

Eclipse seems to solve it similairly (this package runs on the same platform):
https://github.com/NixOS/nixpkgs/blob/6c9e17080423bcba9f09ebbcd5ffc55e34af2449/pkgs/applications/editors/eclipse/build-eclipse.nix#L45

Reply:
Can you give it a try? Changing LD_LIBRARY_PATH affects alls applications started within the program while patchelf is only for the program itself.

Reply:
@SuperSandro2000 
i did, thus the question.
`patchelf` won't be enough, we'd have to patch some of the jar files.

Thinking of it - passing the library to it's childen might actually be the desired behavior - unless we patch third-party plugins too.